k-distribution/k-tutorial/1_basic/05_modules/README.md

```diff
@@ -6,14 +6,14 @@ complete K definition.
 
 ## K's outer syntax
 
-Recall from [Lesson 1.3](../03_parsing/README.md) that K's grammar is broken into
-two components: the **outer syntax** of K and the **inner syntax** of K. Outer
-syntax, as previously mentioned, consists of **requires**, **modules**,
+Recall from [Lesson 1.3](../03_parsing/README.md) that K's grammar is broken
```

Comment:
The added asterisks changed the text alignment, this looks more complicated than it really is.